README.md

```diff
@@ -144,6 +144,22 @@ There are 2 types of Cypress tests, e2e & component.
 
 If you are creating an e2e test, it will live in the `cypress/e2e` directory. Component tests will need to be created in a directory called `cypress/component `
 
+#### Setup your Cypress env variables
+- the Cypress suite requires environment variables that should be stored in your `.env.local` and not committed to git.
+- (these two are not secrets)
+  - TEST_SCIENTIST_USER=test@test.com
+  - TEST_SCIENTIST_PW=!test1234
```

Comment:
since `.env.local` is intended for secrets, can you move these two variables into a `.env.development` file instead?

_I think the CI should still work with them in a dev file, but we'll see._

Reply:
I originally tried to do that, but couldn't get the dot env configuration to work

Then I found this article that says in the dotenv docs they advise against doing that:
https://stackoverflow.com/a/68731401/15106020

Would you like me to move them to the regular .env file though since they are not secrets?

Reply:
I went ahead and moved them to the regular env file, and updated the readme

Reply:
that SO post appears to be outdated. I was referencing the [next js docs](https://nextjs.org/docs/basic-features/environment-variables#default-environment-variables) for adding the `.env.development`.

either way, if `.env.development` didn't work, then `.env` is fine.

![image](https://user-images.githubusercontent.com/29032869/219433835-481143ed-4a13-441c-8ab9-1b53e449e0d6.png)

Reply:
@alishaevn sorry, i meant to attach the dotenv documentation and not the SO post that included said documentation. I did see those next docs which is why i went through trying to add the env.development file in the first place, which was unsuccessful

https://www.npmjs.com/package/dotenv#should-i-have-multiple-env-files

since this is using `dotenv` to load the environment files, and `dotenv` strongly recommends against multiple env files, that is possibly why it wasn't working